frontend/src/app/shared/community-agreement/community-agreement.widget.html

```diff
@@ -0,0 +1,196 @@
+<mat-card class="community-agreement-dialog" #dialogContainer>
```

Comment:
NIT: Could take this html out of the card so the dialog doesn't appear as a "card within a card".

Reply:
Fixed